connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/SparkConnectClientSuite.scala

```diff
@@ -16,17 +16,79 @@
  */
 package org.apache.spark.sql.connect.client
 
+import java.util.concurrent.TimeUnit
+
+import io.grpc.Server
+import io.grpc.netty.NettyServerBuilder
+import io.grpc.stub.StreamObserver
+import org.scalatest.BeforeAndAfterEach
 import org.scalatest.funsuite.AnyFunSuite // scalastyle:ignore funsuite
```

Comment:
Since we're here, can we avoid using `AnyFunSuite`?

Reply:
+1

Reply:
Sure, updating to `FunSuite` instead

Reply:
Actually, maven complains that `object FunSuite is not a member of package org.scalatest` and I don't see any other place in spark where we use `FunSuite` over `AnyFunSuite`.

Is there an alternative in this case?